jenkins/shared-library/vars/codeCoverage.groovy

```diff
@@ -11,17 +11,26 @@ void uploadCodeCoverage(String flag) {
 	}
 }
 
+void logCodeCoverageMinimum(Integer minimumCodeCoverage) {
+	logger.logInfo("Minimum code coverage is ${minimumCodeCoverage}")
+}
+
 void verifyCodeCoverageResult(String tool, Integer minimumCodeCoverage) {
 	Map codeCoverageCommand = [
-		'coverage': { target ->
+		'coverage': { Integer target ->
+			logCodeCoverageMinimum(target)
 			runScript('coverage xml')
 			runScript("coverage report --fail-under=${target}")
 		},
-		'nyc': { target ->
+		'nyc': { Integer target ->
+			logCodeCoverageMinimum(target)
 			runScript('npx nyc@latest report --lines')
 			runScript("npx nyc@latest check-coverage --lines ${target}")
+		},
+		'jacoco': { Integer target ->
+			logger.logInfo('Minimum code coverage is set pom.xml')
```

Comment:
set *in* pom? should you read from pom and display result here?

Reply:
yea, I was thinking that also.
Or to keep the code coverage consistent across all tools, take the minimumCodeCoverage and overwrite the value in the pom.  Any preference?

Reply:
overwriting is fine too, but in that case would probably put 0 in pom with comment

Reply:
Will read the value and display it here.  Thanks.